extensions/wikia/Chat2/js/views/views.js

```diff
@@ -211,8 +211,9 @@ var UserView = Backbone.View.extend({
 	render: function(){
 		//$().log("ABOUT TO RENDER THIS USER: " + JSON.stringify(this.model));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)